src/amo/components/LandingPage.js

```diff
@@ -108,9 +109,29 @@ export class LandingPageBase extends React.Component {
     }
 
     const { addonType, html } = content;
+    const themeText = 'Change your browser’s appearance. Choose from thousands of themes to give Firefox the look you want.';
+    const extensionsText = 'Install powerful tools that make browsing faster and safer, add-ons make your browser yours.';
 
     return (
       <div className={classNames('LandingPage', `LandingPage-${addonType}`)}>
+
+        <div className="Addon-Header">
+          <Icon name={classNames(`${addonType}`)} className="Addon-Icon" />
+          <div className="Addon-Header-Text">
+            <span className="Addon-Header-Text-Bold">
+              {addonType === 'persona' ? 'Themes' : 'Extensions' }
+            </span>
+            <p className="Addon-Header-Text-Content">
+              {addonType === 'persona' ? themeText : extensionsText }
+            </p>
+          </div>
+        </div>
+
+        <button className="Browse-Button">
```

Comment:
This should link to the category page for this extension type. We have a `Link` component in `amo/components/Link` and this should use that. There are styles for the white/blue colour scheme in `Home.scss`.

Ideally the common styles would move into `Link` and this colour scheme can be chosen with something like `<Link appearance="light">...</Link>`. Since `Link` uses the `@include button()` mixin you should be able to use the colour from the homepage link and set a class like `Link--light` that will use the mixin as `@include button(#fff, $link-color)`.